docs/content/users/extend/additional-services.md

```diff
@@ -131,10 +132,10 @@ Go templating resources:
 
 ## Additional services in ddev-contrib
 
-Commonly-used services will be migrated from the [ddev-contrib](https://github.com/ddev/ddev-contrib) repository to individual, tested, supported repositories, but the repository still has a wealth of additional examples and instructions:
+Commonly-used services are being migrated from the [ddev-contrib](https://github.com/ddev/ddev-contrib) repository to individual, tested, supported add-on repositories, but the repository still has a wealth of additional examples and instructions:
 
 * **Old PHP Versions to Run Old Sites**: See [Old PHP Versions](https://github.com/ddev/ddev-contrib/blob/master/docker-compose-services/old_php)
 * **RabbitMQ**: See [RabbitMQ](https://github.com/ddev/ddev-contrib/blob/master/docker-compose-services/rabbitmq)
 * **TYPO3 Solr Integration**: See [TYPO3 Solr](https://github.com/ddev/ddev-contrib/blob/master/docker-compose-services/typo3-solr)
 
-Your pull requests to integrate other services are welcome at [ddev-contrib](https://github.com/ddev/ddev-contrib).
+Your pull requests to integrate other services are welcome at [ddev-contrib](https://github.com/ddev/ddev-contrib), but creating a supported add-on is preferred.
```

Comment:
[Use friendlier active third person](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#use-active-third-person) and attempt to explain _why_ we’d prefer a supported add-on.

```suggestion
While we welcome requests to integrate other services at [ddev-contrib](https://github.com/ddev/ddev-contrib), we encourage creating a supported add-on that’s more beneficial to the community.
```